fe/fe-core/src/main/java/com/starrocks/credential/CloudConfigurationConstants.java

```diff
@@ -83,21 +80,35 @@ public class CloudConfigurationConstants {
 
     // Credential for Google Cloud Platform (GCP)
     // For Google Cloud Storage (GCS)
-    public static final String GCP_GCS_USE_COMPUTE_ENGINE_SERVICE_ACCOUNT = "gcp.gcs.use_compute_engine_service_account";
+    public static final String GCP_GCS_USE_COMPUTE_ENGINE_SERVICE_ACCOUNT =
+            "gcp.gcs.use_compute_engine_service_account";
     public static final String GCP_GCS_SERVICE_ACCOUNT_EMAIL = "gcp.gcs.service_account_email";
     public static final String GCP_GCS_SERVICE_ACCOUNT_PRIVATE_KEY = "gcp.gcs.service_account_private_key";
     public static final String GCP_GCS_SERVICE_ACCOUNT_PRIVATE_KEY_ID = "gcp.gcs.service_account_private_key_id";
     public static final String GCP_GCS_SERVICE_ACCOUNT_IMPERSONATION_SERVICE_ACCOUNT =
             "gcp.gcs.impersonation_service_account";
 
     // Credential for HDFS
-    // TODO: Refactor the name of parameters
     public static final String HDFS_AUTHENTICATION = "hadoop.security.authentication";
-    public static final String HDFS_USER_NAME = "username";
-    public static final String HDFS_PASSWORD = "password";
-    public static final String HDFS_KERBEROS_PRINCIPAL = "kerberos_principal";
-    public static final String HDFS_KERBEROS_KEYTAB = "kerberos_keytab";
-    public static final String HDFS_KERBEROS_KEYTAB_CONTENT = "kerberos_keytab_content";
+    @Deprecated
+    public static final String HDFS_USERNAME_DEPRECATED = "username";
+    public static final String HDFS_USERNAME = "hadoop.username";
+    @Deprecated
+    public static final String HDFS_PASSWORD_DEPRECATED = "password";
+    public static final String HDFS_PASSWORD = "hadoop.password";
+    public static final String HDFS_KERBEROS_PRINCIPAL_DEPRECATED = "kerberos_principal";
+    public static final String HDFS_KERBEROS_PRINCIPAL = "hadoop.kerberos.principal";
+    @Deprecated
+    public static final String HDFS_KERBEROS_KEYTAB_DEPRECATED = "kerberos_keytab";
+    public static final String HADOOP_KERBEROS_KEYTAB = "hadoop.kerberos.keytab";
+
+    @Deprecated
+    public static final String HDFS_KERBEROS_KEYTAB_CONTENT_DEPRECATED = "kerberos_keytab_content";
+    public static final String HADOOP_KERBEROS_KEYTAB_CONTENT = "hadoop.kerberos.keytab_content";
+    public static final String HDFS_CONFIG_RESOURCES = "hadoop.config.resources";
+    public static final String HDFS_CONFIG_RESOURCES_LOADED = "hadoop.config.resources.loaded";
```

Comment:
add some comment to explain `HDFS_CONFIG_RESOURCES_LOADED`

Reply:
OK. Will fix that.